test/e2e/user_management_test.go

```diff
@@ -128,6 +128,55 @@ func (s *userManagementTestSuite) TestUserDeactivation() {
 		})
 	})
 
+	s.T().Run("tests for basic tier with deactivation disabled", func(t *testing.T) {
```

Comment:
hmm... shouldn't it be rather in `nstemplatetier_test.go` file - we want to verify the functionality of the new tier, right?

Reply:
Not sure, I think you could argue it both ways. The way I thought about it is tiers are more like part of the configuration, the tiers contain values we can configure but the actual behaviour is handled by the controllers that react on those values. So I figured the tier tests validate that the values we're setting are within some expected parameters and the other tests are testing the functionality. So for user management which includes Banning, Deactivating and Disabling users I figured it made sense to test the automatic deactivation here as well.

Reply:
ok, fair enough :+1: keep it here :-)